packages/documentation/docs/integration/flow-of-funds.md

```diff
@@ -0,0 +1,119 @@
+# Flow of funds
+
+Rafiki communicates with the Account Servicing Entity over webhook events, which are defined in the _Getting Started_ section. Each event requires the Account Servicing Entity to either deposit or withdraw liquidity into or from Rafiki. This document will provide an example flow for each of the defines webhook events.
+
+## `incoming_payment.created`
+
+The `incoming_payment.created` event indicates that an incoming payment has been created. It has not yet received any funds so no action is required.
```

Comment:
Should we mention here that (for example) you could notify a user of an upcoming incoming payment?